### ISSUE TITLE
Is the eye's obj file of boggie correct?

### ISSUE THREAD

**Issue by Bestoa** [ssloy/tinyrenderer#75]
Hi,

I try to render model boggie, but I find the eyes of the render result is not correct.
![eye](https://user-images.githubusercontent.com/5378112/115100075-d1330c00-9f6c-11eb-89ee-d921552dcb19.png)

![eye_head](https://user-images.githubusercontent.com/5378112/115100063-bf516900-9f6c-11eb-8d9b-973fe1d59eaf.png)

Thanks
Ming Fang

**Comment by Bestoa** [comment]
![Screenshot from 2021-04-19 11-24-28](https://user-images.githubusercontent.com/5378112/115177519-fd33c600-a101-11eb-89ba-ba4ebbd22401.png)
I also export this model into Blender, seems the UV map is wrong:-(

**Comment by ssloy** [comment]
Yeah, I am too lazy to fix the issue, the eyses are made of two shells, the inner shell is okay with the texture, and the outer shell should have another semi-transparent texture. Remove the outer shell and you are good.

![image](https://user-images.githubusercontent.com/1029298/115232454-00857c80-a117-11eb-9e34-b9c9892d9a20.png)
